compliance_checker/cf/cf.py

```diff
@@ -723,9 +725,9 @@ def check_ancillary_data(self, ds):
             anc_result = Result(BaseCheck.HIGH, name=('§3.4 Ancillary Variables', k))
             msgs = []
 
-            if not isinstance(anc, str):
+            if not isinstance(anc, basestring):
```

Comment:
If you look at https://github.com/DanielJMaher/compliance-checker/blob/ancillary/compliance_checker/cf/cf.py#L12-L15 it catches if it is invalid and then defaults it to str  

This is per Kyle's guidance.  It should be fine in Py3